src/ucs/config/parser.c

```diff
@@ -1095,16 +1098,16 @@ int ucs_config_sscanf_key_value(const char *buf, void *dest, const void *arg)
     size_t buf_len                            = strlen(buf) + 1;
     const char *TOKEN_DELIM                   = ",";
     const char *VALUE_DELIM                   = ":";
-    const char *common_value                  = NULL;
-    size_t key_count, index;
-    uint8_t *key_set;
+    const char *default_value                 = NULL;
+    const char *key, *value, **values;
     char *token, *tokens;
-    const char *key, *value;
-    int key_index;
+    size_t key_count;
```

Comment:
the name 'index' is not recommended since it's a name of C library function

Reply:
sure